src/components/ChattingList/ChattingListView.js

```diff
@@ -0,0 +1,32 @@
+import React, { useState } from "react";
+import ChattingProfile from "./ChattingProfile";
+import ChattingListTopBar from "./ChattingListTopBar";
+import userProfileSet from "../../data/User";
+import styled from "styled-components";
+
+const Margin = styled.div`
+  height: 1.9vh;
+`;
+
+function ChattingList() {
+  const [matchUserSet, setMatchUserSet] = useState(userProfileSet);
+
+  function searchUser(searchUserInput) {
+    setMatchUserSet([]);
+    for (let userProfile of userProfileSet) {
+      if (userProfile.name.includes(searchUserInput)) {
+        setMatchUserSet((matchUser) => [...matchUser, userProfile]);
+      }
+    }
```

Comment:
```suggestion
   function searchUser(searchUserInput) {
    const matchedProfiles = userProfileData.filter(userProfile => 
      userProfile.name.includes(searchUserInput)
    );
    setFilteredUserProfiles(matchedProfiles);
  }
```
이렇게 하면  `setMatchUserSet()` 함수를 2번 호출하지 않고 한번에 업데이트 할 수 있어요!